markup.go

```diff
@@ -170,7 +170,19 @@ func If(cond bool, markup ...MarkupOrComponentOrHTML) MarkupOrComponentOrHTML {
 	if cond {
 		return List(markup)
 	}
-	return nil
+	var result []MarkupOrComponentOrHTML
+	for _, m := range markup {
+		if m == nil {
+			continue
```

Comment:
The curly part is when the first element in the slice is a `nil`, since we don't know whether we're dealing with `Component`/`HTML`, or with `Markup`.  I suppose the correct thing to do might be to iterate over the whole slice before doing anything - if we encounter any `Markup`, return `nil`, otherwise just return a slice of `Tag("noscript")` with the same length as the argument, though I'm not sure what the correct behaviour should be for a list of `nil`s.